packages/forms/src/Components/Concerns/HasHint.php

```diff
@@ -3,15 +3,16 @@
 namespace Filament\Forms\Components\Concerns;
 
 use Closure;
+use Filament\Forms\Components\Actions\Action;
 use Illuminate\Support\HtmlString;
 
 trait HasHint
 {
-    protected string | HtmlString | Closure | null $hint = null;
+    protected string | HtmlString | Closure | Action | null $hint = null;
```

Comment:
I think it makes sense to move `Action` before `Closure` here (also in the other places).

```suggestion
    protected string | HtmlString | Action | Closure | null $hint = null;
```